sdk/cognitiveservices/textanalytics/Azure.AI.TextAnalytics/src/TextAnalyticsClient.cs

```diff
@@ -22,6 +22,8 @@ public partial class TextAnalyticsClient
         private readonly string _apiVersion;
         private readonly TextAnalyticsClientOptions _options;
 
+        private readonly string DefaultCognitiveScope = "https://cognitiveservices.azure.com/.default";
```

Comment:
Is it always one and the same scope, or it can be similar to the AzConfig?

Reply:
I believe it is always the same scope.